arkhn/pyrog
# fix: bad request when editing a resource

## Fixes


Fixes #371 by @elsiehoffet-94 
## Description

* Remove id from sqlColumn field in Filter type definition => doesn't exist in graphql schema
* Remove __typename attr from filters attr before updating a resource => see related issue

## Technical details

Related issue: https://github.com/apollographql/apollo-client/issues/1913

## Tests

1. Select a source
2. Select a resource which has filters
3. Edit the resource
4. Edit the label or the filter and save. It shouldn't return an error 400 anymore and the fields should be correctly updated.
